app/helpers/application_helper.rb

```diff
@@ -1,17 +1,17 @@
 module ApplicationHelper
   def thumbnail_url(image)
-    if image.is_video?
+    if image&.is_video?
```

Comment:
these are not necessary but helps while developing not getting errors when testing and it doesn't harm